src/Renderer/LineMaterial.js

```diff
@@ -0,0 +1,44 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+
+import * as THREE from 'three';
+import BasicMaterial from './BasicMaterial';
+import Fetcher from '../Core/Commander/Providers/Fetcher';
+import LineVS from './Shader/LineVS.glsl';
+import LineFS from './Shader/LineFS.glsl';
+
+const LineMaterial = function LineMaterial(options) {
+    BasicMaterial.call(this);
+
+    if (options === undefined) {
+        throw new Error('options is required');
+    }
+
+    this.fragmentShader = this.fragmentShaderHeader + LineFS;
+    this.vertexShader = this.vertexShaderHeader + LineVS;
+
+    this.wireframe = false;
+
+    const texture = options.texture ? Fetcher.texture.load(options.texture) : undefined;
+
+    this.uniforms.THICKNESS = { value: options.linewidth };
+    this.uniforms.MITER_LIMIT = { value: true };
+    this.uniforms.WIN_SCALE = { value: new THREE.Vector2(window.innerWidth, window.innerHeight) };
```

Comment:
This code is wrong (window size might be different of itowns container size).
And the container size can change so this uniforms probably needs to be updated before rendering.
(this is'nt something you'll want to do in this PR though)